overlay.d/05core/usr/lib/dracut/modules.d/35coreos-ignition/coreos-post-ignition-checks.sh

```diff
@@ -0,0 +1,18 @@
+#!/bin/bash
+# See coreos-post-ignition-checks.service for more information about this script
+set -euo pipefail
+
+# Get mountpoint of boot device
+bootmnt=$(findmnt --source /dev/disk/by-label/boot -no TARGET; 2>/dev/null)
+if [ "$bootmnt" = "" ]; then
```

Comment:
```suggestion
if [ -z "$bootmnt" ]; then
```

Another way to do this..